en/services/gimbal_v2.md

```diff
@@ -0,0 +1,282 @@
+
+### Gimbal manager deconfliction rules
+
+It is possible for multiple components to want to control a gimbal at the same time: e.g a ground station, a companion computer, or the autopilot running a mission.
+This can create situations where the gimbal would receive conflicting messages from the different components, and hence may behave unintendedly or unexpectedly from the viewpoint of one of the components (and ultimately the user). Thus, decision making is required in order to establish proper and predictable gimbal operation.
+
+The deconfliction of various inputs is the task of the gimbal manager. The gimbal manager should implement the rules below:
+
+1. If an attitude has been set using a command ([DO_GIMBAL_MANAGER_ATTITUDE](#MAV_CMD_DO_GIMBAL_MANAGER_ATTITUDE), [DO_SET_ROI_LOCATION](#MAV_CMD_DO_SET_ROI_LOCATION), [DO_GIMBAL_MANAGER_TRACK_POINT](#MAV_CMD_DO_GIMBAL_MANAGER_TRACK_POINT), or [DO_GIMBAL_MANAGER_TRACK_RECTANGLE](#MAV_CMD_DO_GIMBAL_MANAGER_TRACK_RECTANGLE) it takes precedence over any other input until a [DO_GIMBAL_MANAGER_ATTITUDE](#MAV_CMD_DO_GIMBAL_MANAGER_ATTITUDE) with [GIMBAL_MANAGER_FLAGS_NONE](#GIMBAL_MODE_NONE) or a [DO_SET_ROI_NONE](#MAV_CMD_DO_SET_ROI_NONE) is set.
+   Commands will interfere with each other, whichever command is received last takes precedence.
+2. A gimbal angle or tracking location initiated by a command can be nudged by [GIMBAL_MANAGER_SET_ATTITUDE](#GIMBAL_MANAGER_SET_ATTITUDE) if the "nudge bit" is set.
+3. A gimbal angle or tracking location initiated by a command can be overridden by [GIMBAL_MANAGER_SET_ATTITUDE](#GIMBAL_MANAGER_SET_ATTITUDE) if the "override bit" is set.
+
+#### Nudging
+
+Nudging is a slight deflection/change on top of the set gimbal attitude:
+ROI: while set at ROI, the gimbal can be moved around to look around. If the sticks are let go, it will snap to the new selected ROI if supported and otherwise snap back to the previous one.
+TRACK: while tracking a point or rectangle, nudging will move the gimbal angle only as much as so that the tracked object is still on the image (otherwise it would lose the tracking). If the sticks are let go, it will snap back to have the tracked object centered.
+ATTITUDE: nudging allows deflection of the angle. If the sticks are let go it will go back to the set attitude.
+
+#### Overriding
+Overriding means that anything set is ignored. It is generally discouraged to use this unless something doesn’t work as intended, a mission is set-up incorrectly, etc..
+
+
+### Custom gimbal device settings
+
+Custom gimbal settings can be accomplished using the component information microservice which is based on a [component definition file](../services/component_def.md) similar to the [camera definition file](../services/camera_def.md).
```

Comment:
@julianoes The component definition file does not exist. Where is this defined?

Reply:
Yes, todo :)